components/script/dom/domrectlist.rs

```diff
@@ -57,3 +66,18 @@ impl DOMRectListMethods for DOMRectList {
         self.Item(index)
     }
 }
+
+impl<'a> Iterator for Iter<'a> {
```

Comment:
`Iter` stores the current position, but perhaps it would be better to ranem `Iter` to `IterableDOMRectList` or `DOMRectListIterator`